README.md

```diff
@@ -1,6 +1,6 @@
 # Cylc Documentation
 
-[![Build Status](https://travis-ci.org/cylc/cylc-doc.svg?branch=master)](https://travis-ci.org/cylc/cylc-doc)
+![](https://github.com/cylc/cylc-doc/workflows/build/badge.svg)
```

Comment:
I get a 404 on that URL.

Reply:
`![test](https://github.com/cylc/cylc-doc/workflows/test/badge.svg)` is what you would be after. However, this shows the status for the latest PR - is that really desired? Wouldn't we want the status on master instead? If so, it can be done easily by adding an `on` trigger to the GH actions workflow